Source/Controllers/DataControllers/SPTableData.m

```diff
@@ -929,18 +931,25 @@ - (NSDictionary *) informationForView:(NSString *)viewName
  */
 - (BOOL)updateStatusInformationForCurrentTable
 {
+
+    SPLog(@"updateStatusInformationForCurrentTable called");
+
 	pthread_mutex_lock(&dataProcessingLock);
 
 	BOOL changeEncoding = ![[mySQLConnection encoding] hasPrefix:@"utf8"];
 
 	// Catch unselected tables and return false
 	if (![tableListInstance tableName]) {
 		pthread_mutex_unlock(&dataProcessingLock);
+        SPLog(@"No table selected, returning");
+        CLS_LOG(@"No table selected, returning");
 		return NO;
 	}
 
 	// Ensure queries are run as UTF8mb4
 	if (changeEncoding) {
+        SPLog(@"changeEncoding to utf8mb4");
+        CLS_LOG(@"changeEncoding to utf8mb4");
```

Comment:
As these couple things get called all over the place, maybe want to clarify the log statement a little more. Or, if we want to know every time encoding changes, could log directly from the setEncoding method instead?

Reply:
Just want to know at this point I think. The macro adds the func and line. I'll double check the context.